.github/workflows/changes.yml

```diff
@@ -225,116 +227,152 @@ jobs:
             - "src/sinks/aws_sqs/**"
             - "src/sinks/util/**"
             - "src/transforms/aws*"
+            - "scripts/integration/aws/**"
           axiom:
             - "src/sinks/axiom.rs"
             - "src/sinks/util/**"
+            - "scripts/integration/axiom/**"
           azure:
             - "src/sinks/azure_**"
             - "src/sinks/util/**"
+            - "scripts/integration/azure/**"
           clickhouse:
             - "src/sinks/clickhouse/**"
             - "src/sinks/util/**"
+            - "scripts/integration/clickhouse/**"
           databend:
             - "src/sinks/databend/**"
             - "src/sinks/util/**"
+            - "scripts/integration/databend/**"
           datadog:
             - "src/common/datadog.rs"
             - "src/internal_events/datadog_*"
             - "src/sources/datadog_agent/**"
             - "src/sinks/datadog/**"
             - "src/sinks/datadog_archives.rs"
             - "src/sinks/util/**"
+            - "scripts/integration/datadog-agent/**"
+            - "scripts/integration/datadog-logs/**"
+            - "scripts/integration/datadog-metrics/**"
+            - "scripts/integration/datadog-traces/**"
+          dnstap:
+            - "src/internal_events/dnstap.rs"
+            - "src/sources/dnstap/**"
+            - "scripts/integration/dnstap/**"
```

Comment:
👀

Reply:
👀 were we not triggering `dnstap` tests individually?

Reply:
The dnstap tests were getting run as part of the merge queue checks, but not in PRs based on file detection. Probably went unnoticed.